pfs/lineLists/skyLines.txt

```diff
@@ -104,38 +104,38 @@
 603.58851            1.00    OH           1 8-2_P1f(7.5)   4
 606.88269            1.00    OH           1 8-2_P1e(8.5)   4
 607.03643            1.00    OH           1 8-2_P1f(8.5)   4
-613.82390            0.13    OH           1 5-0_R1f(4.5)  12
-613.82520            0.13    OH           1 5-0_R1f(4.5)|5-0_R1e(4.5)  12
-613.82650            0.13    OH           1 5-0_R1e(4.5)  12
-613.84660            0.34    OH           1 5-0_R1f(3.5)  12
-613.84840            0.34    OH           1 5-0_R1f(3.5)|5-0_R1e(3.5)  12
-613.85020            0.34    OH           1 5-0_R1e(3.5)  12
-613.93090            0.06    OH           1 UNKNOWN   8
-613.93330            0.07    OH           1 UNKNOWN|UNKNOWN  12
-613.93570            0.06    OH           1 UNKNOWN   8
+613.82390            0.13    OH         257 5-0_R1f(4.5)  12
```

Comment:
So is this undoing the problem with the last commit? But then what does the commit message mean? In any case, it should be squashed with the previous commit.